src/pevm.rs

```diff
@@ -172,7 +172,8 @@ impl Pevm {
         thread::scope(|scope| {
             for _ in 0..concurrency_level.into() {
                 scope.spawn(|| {
-                    let mut task = scheduler.next_task();
+                    let mut task_none_counter = 0;
```

Comment:
We can move this to the beginning of `next_task`, right before `while !self.aborted.load(Ordering::Acquire) {`?